cmf-cli/resources/template_feed/help/from1000/Help.Package/package.json

```diff
@@ -18,17 +18,16 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^15.2.1",
-    "@angular/common": "^15.2.1",
-    "@angular/compiler": "^15.2.1",
-    "@angular/core": "^15.2.1",
-    "@angular/forms": "^15.2.1",
-    "@angular/localize": "^15.2.1",
-    "@angular/platform-browser": "^15.2.1",
-    "@angular/platform-browser-dynamic": "^15.2.1",
-    "@angular/router": "^15.2.1",
-    "@angular/service-worker": "^15.2.1",
-    "@criticalmanufacturing/ngx-schematics": "<%= $CLI_PARAM_NGXSchematicsVersion %>",
```

Comment:
Was this lost between changes or was it intentionally converted to a dev dep?

Reply:
it's intentional, schematics aren't required for runtime